keepassxc-browser/global.js

```diff
@@ -4,9 +4,9 @@ const IGNORE_NOTHING = 'ignoreNothing';
 const IGNORE_NORMAL = 'ignoreNormal';
 const IGNORE_FULL = 'ignoreFull';
 
-var schemeSegment = '(\\*|http|https|ws|wss|file|ftp)';
-var hostSegment = '(\\*|(?:\\*\\.)?(?:[^/*]+))?';
-var pathSegment = '(.*)';
+const schemeSegment = '(\\*|http|https|ws|wss|file|ftp)';
+const hostSegment = '(\\*|(?:\\*\\.)?(?:[^/*]+))?';
+const pathSegment = '(.*)';
 
 var isFirefox = function() {
```

Comment:
Oh, hey, this is not really needed. We are not in ugly websites – we have proper APIs for that. :laughing: 

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getBrowserInfo

(Okay, just see browser support is not yet that good, but hey… I assume other browsers implement it too. You may just fallback or so.)